### PR TITLE
 Solution for Array Manipulation

### PR DESCRIPTION
**Pull request by voldemortuk**

### CODE REVIEW

File: Data Structures/Arrays/ArrayManipulation/Solution.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    {
+    long[] computation = new long[n];
+
+    for (int i = 0; i < queries.length; i++) 
+    {
+      int a = queries[i][0] - 1;
+      int b = queries[i][1] - 1;
+      int k = queries[i][2];
+
+      computation[a] + = k;
```
Comment (Kvaibhav01) [inline Data Structures/Arrays/ArrayManipulation/Solution.java]: This expression is illegal.

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
1. Make sure to use braces like this:

```
static long arrayManipulation(int n, int[][] queries) {
    long[] computation = new long[n];
```

Instead of this:

```
static long arrayManipulation(int n, int[][] queries)
    {
    long[] computation = new long[n];
```

2. Change path from _Data Structures/Arrays/ArrayManipulation/Solution.java_ to _Data Structures/Arrays/**Array Manipulation**/Solution.java_.

File: Data Structures/Arrays/Array Manipulation/Solution.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    long max = 0;  long sum = 0;
+    
+    for (int i = 0; i < n; i++) 
+    {
+      sum += computation[i];
+      max = Math.max(max, sum);
+    }
+
+    return max;
+  }
```
Comment (Kvaibhav01) [inline Data Structures/Arrays/Array Manipulation/Solution.java]: Please reformat the code. Don't use braces in the next line anywhere.

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]

File: Data Structures/Arrays/ArrayManipulation/Solution.java (hunk 1 above)
Comment (voldemortuk) [reply Data Structures/Arrays/ArrayManipulation/Solution.java]: okay I'll get onto it

File: Data Structures/Arrays/Array Manipulation/Solution.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import java.text.*;
+import java.util.*;
+import java.util.concurrent.*;
+import java.util.regex.*;
+
+public class Solution {
+
+    // Complete the arrayManipulation function below.
+    
+    static long arrayManipulation(int n, int[][] queries){
```
Comment (Kvaibhav01) [inline Data Structures/Arrays/Array Manipulation/Solution.java]: Give space.

```suggestion
    static long arrayManipulation(int n, int[][] queries) {
```

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Check the suggested change and I'll merge the PR.

**Comment by voldemortuk** [comment]
> Check the suggested change and I'll merge the PR.

yeah I have done it

**Review by Kvaibhav01 — APPROVED** [review]

**Comment by voldemortuk** [comment]
great

**Pull request merged**